lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+module Sisimai::Bite::Email
+  module Facebook
+    # Sisimai::Bite::Email::Facebook parses a bounce email which created by Facebook.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Facebook.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\AFacebook [<]mailer-daemon[@]mx[.]facebook[.]com[>]\z/,
+        :subject => %r/\ASorry, your message could not be delivered\z/,
+      }
+      Re1 = {
+        :begin   => %r/\AThis message was created automatically by Facebook[.]\z/,
+        :rfc822  => %r/\AContent-Disposition: inline\z/,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+
+      # http://postmaster.facebook.com/response_codes
+      # NOT TESTD EXCEPT RCP-P2
+      ReFailure = {
+        userunknown: [
+          'RCP-P1', # The attempted recipient address does not exist.
+          'INT-P1', # The attempted recipient address does not exist.
+          'INT-P3', # The attempted recpient group address does not exist.
+          'INT-P4', # The attempted recipient address does not exist.
+        ],
+        filtered: [
+          'RCP-P2', # The attempted recipient's preferences prevent messages from being delivered.
+          'RCP-P3', # The attempted recipient's privacy settings blocked the delivery.
+        ],
+        mesgtoobig: [
+          'MSG-P1', # The message exceeds Facebook's maximum allowed size.
+          'INT-P2', # The message exceeds Facebook's maximum allowed size.
+        ],
+        contenterror: [
+          'MSG-P2', # The message contains an attachment type that Facebook does not accept.
+          'MSG-P3', # The message contains multiple instances of a header field that can only be present once. Please see RFC 5322, section 3.6 for more information
```

Comment:
__[rubocop]__
* Metrics/LineLength: Line is too long. [164/144] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->